### PR TITLE
fix(api): Fixing dev url for auth-public-api

### PR DESCRIPTION
**Pull request by saevarma**
## What

Changing the `auth-public-api` to be operated on `/api` path, so configuring the `api` infra to update url.

## Why

So the service doesn't care about any prefixes and open for the `/swagger` url to expose the OpenAPI document of the service.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/api/infra/api.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ export const serviceSetup = (services: {
         prod: 'island-is-prod-upload-api',
       },
       AUTH_PUBLIC_API_URL: {
-        dev: 'https://identity-server.dev01.devland.is',
+        dev: 'https://identity-server.dev01.devland.is/public',
```
Comment (sindrig) [inline apps/api/infra/api.ts:43]: why only for dev and didn't you forget to run `yarn charts`?

Comment (saevarma) [reply apps/api/infra/api.ts:43]: Yes I forgot to run yarn charts.

I'm testing the ingress rewrite-target rule on dev by manually configuring in k9s.
Then the plan is to change this path to `/api` and do the corresponding changes in the Helm repo for the identity-server apis.

Comment (sindrig) [reply apps/api/infra/api.ts:43]: ah right so this is work in progress. you know you can create draft pull requests, and then there will be no reviewers. kind of nice when you're playing a :ninja:

**Review by pshomov — APPROVED** [review]
LGTM

**Pull request merged**